test/Interop/Cxx/ergonomics/swift-bridging-annotations.swift

```diff
@@ -110,6 +129,26 @@ private:
 // CHECK:   class func getInstance() -> LoggerSingleton!
 // CHECK: }
 
+// CHECK-LABEL: func takeLoggersByPointer(
+// CHECK-SAME: _ ptr: LoggerSingleton!,
+// CHECK-SAME: _ ptr_ptr: UnsafeMutablePointer<LoggerSingleton?>!,
+// CHECK-SAME: _ ptr_ptr_ptr: UnsafeMutablePointer<UnsafeMutablePointer<LoggerSingleton?>?>!)
+
+// CHECK-LABEL: func takeLoggersByReference(
+// CHECK-SAME: _ ref: LoggerSingleton,
+// CHECK-SAME: _ ref_ptr: inout LoggerSingleton!,
+// CHECK-SAME: _ ref_ptr_ptr: inout UnsafeMutablePointer<LoggerSingleton?>!)
+
+// CHECK-LABEL: func takeLoggersByConstPointer(
+// CHECK-SAME: _ pointee0: UnsafeMutablePointer<LoggerSingleton?>!,
+// CHECK-SAME: _ pointee1: UnsafeMutablePointer<LoggerSingleton?>!,
+// CHECK-SAME: _ pointer: UnsafePointer<LoggerSingleton?>!)
+
+// CHECK-LABEL: func takeLoggersByConstReference(
+// CHECK-SAME: _ pointee0: inout LoggerSingleton!,
+// CHECK-SAME: _ pointee1: inout LoggerSingleton!,
+// CHECK-SAME: _ pointer: LoggerSingleton!)
```

Comment:
This seems correct to me and matches the way we hide the `const&` indirection for value types.